stock_quantity_history_location/models/stock_quant.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from odoo import api, models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    @api.model
+    def get_views(self, views, options=None):
+        res = super().get_views(views, options=options)
```

Comment:
@AnaisLForgeFlow Maybe a good improvement should be adding a little test calling the view.

Reply:
@rousseldenis sure, we can add this for better test coverage

Reply:
@rousseldenis I've just added the test case, I believe it should be ready to merge 🙂